src/domain/communication/messaging/DirectMessaging/DirectMessageDialog.tsx

```diff
@@ -87,8 +88,9 @@ export const DirectMessageDialog: FC<MessageUserDialogProps> = ({
               <Form noValidate autoComplete="off">
                 <GridContainer disablePadding marginBottom={gutters(1)}>
                   <GridProvider columns={breakpoint === 'xs' ? GRID_COLUMNS_MOBILE : GRID_COLUMNS_DESKTOP} force>
-                    {messageReceivers?.map(receiver => (
+                    {messageReceivers?.map((receiver, index) => (
                       <ProfileChip
+                        key={receiver.id ?? index}
```

Comment:
Yes, if I make it mandatory I'm going to have to fix PorfileBanner. maybe there is a better way to do this, but at the end .id can be undefined because all the rest of the properties can be undefined too